### DISCUSSION TITLE
How to transform images into vectors?

### DISCUSSION THREAD

**Discussion by Pean-Mura** [milvus-io/milvus#5507] [Q&A and General discussion]
Hello,I see you use VGG16 to get the vectors of images, but which layer is the best? If remove the FC layers, there will be 25088 dims. It will be a large size. But try get the fc1 layer, which is 4096 dims. I can't get a good result. what should I do?

**Comment by shiyu22** [comment]
In fact, we are not familiar with training models, Milvus is used for vector search. And we usually use pre-trained models, you can refer to [Bootcamp about image search](https://github.com/milvus-io/bootcamp/tree/master/solutions/image_similarity_search), which used Keras' ResNet50 model.

**Reply by Pean-Mura** [reply]
Okay. I just can't find a better way to extract features. Thanks

**Reply by shiyu22** [reply]
Maybe you can get the answer in communities like TensorFlow.
